xbmc/network/httprequesthandler/HTTPImageTransformationHandler.cpp

```diff
@@ -35,12 +35,11 @@ CHTTPImageTransformationHandler::CHTTPImageTransformationHandler()
 
 CHTTPImageTransformationHandler::CHTTPImageTransformationHandler(const HTTPRequest &request)
   : IHTTPRequestHandler(request),
-    m_url(),
+    m_url(m_request.pathUrl.substr(ImageBasePath.size())),
```

Comment:
for readability imo this was better was where before

Reply:
Nope. The change is just fine. Readability is not the point here. It's performance.